test/Program.cs

```diff
@@ -53,11 +60,82 @@ public static double RankineToKelvin(double temp)
         {
             return CelsjusToKelvin(RankineToCelsjus(temp));
         }
+
+
         static void Main(string[] args)
         {
-            double tem;
-            tem = FarenhaitToCelsjus(10);            
-            Console.WriteLine("wynik "+tem);
+            Char TemperatureImputType = 'C';
+            Double TemperatureImputValue = 0;
+            do
+            {
+                Console.WriteLine("wybierz jednostek temperatury : \n" +
+                    "C - Celsjus \n" +
+                    "F - Farenhait \n" +
+                    "K - Kelvin \n" +
+                    "R - Ten co pierwsze slysze XD \n:"
+                    );
+
+                TemperatureImputType = Console.ReadLine()[0];
+
+            } while (   TemperatureImputType != 'C' && 
```

Comment:
Do use UpperCase function on LowerCase function to reduce number of checks in this while.

Reply:
Even better solution. Prepare a list of viable inputs and use Contains method of the list to check whether TemperatureInputType is in the list.